Form1.cs

```diff
@@ -103,14 +103,14 @@ public partial class Form1 : Form {
 
 			File.WriteAllText(p, acf);
 
-			MessageBox.Show("Patch applied. If everything worked correctly Steam should most likely not ask you to update Beat Saber any more. If it still does its probably related to DLC(Assmung you have any) and should not update the game, just to be sure I advise you to backup your game version", "Success");
+			MessageBox.Show("Patch applied. If everything worked out correctly, Steam should most likely not ask you to update Beat Saber anymore. If it still does, this may be related to DLC (assuming you have any) and should not update the game. Be sure to backup your game version.", "Success");
```

Comment:
in 99% of cases backing up the game manually isnt needed, this makes it sound like one should always do it